languages/en.ini

```diff
@@ -922,6 +932,10 @@ note_785_7 = "Merged with / Forms"
 note_785_8 = "Changed back to"
 note_787 = "Other relationship"
 Notes = "Notes"
+Notifications Broadcasts = "Notifications Broadcasts"
+Notifications Pages = "Notifications Pages"
+notifications_content = "Content (<a href="https://www.markdownguide.org/basic-syntax/" target="_blank">Markdown</a>, <a href="https://packagist.org/packages/elgigi/commonmark-emoji" target="_blank">Emojis</a>)"
```

Comment:
I added `_html` to the translation key. Until now I planned to make those pages translatable, but must admin I have never thought about the admin area being English-only. It might be a nice for the intended users to use these pages  in German but I don't think that this is a must.

Reply:
There's no reason that the entire admin area has to be English-only; it's just been kept that way so far to reduce the load on our volunteer translators, since internationalizing public pages is a higher priority than internationalizing internal ones. And my suggestion of moving the links to a static content page doesn't prevent internationalization either; you can have different versions of the content for different languages by using different suffixes on the content files. I just thought that it would be easier in the future to make changes to a content page than to potentially have to update URLs in all of the language files.

Reply:
Ah, ok! I did not directly understand that this was primarily about the link to the external markdown page. I will look into the `content/markdown_en.phtml` solution.

Reply:
`content/markdown.phtml` exists already, but I'm unsure what it does. Would a generell help text about markdown in `content/markdown_en.phtml` overwrite the functionality in `content/markdown.phtml`?

Reply:
Sorry, I forgot that `content/markdown.phtml` exists to actually render content written in Markdown. We should not create a new template that would conflict with that, as it would be very confusing. I think if you revise my suggestion to use `content/markdown-help_en.phtml` or something similar, that would be better. :-)

Reply:
Also, for general information about how static content works, you might find the [Static Content wiki page](https://vufind.org/wiki/configuration:static_pages) helpful.

Reply:
Thanks for clarifying what  `content/markdown.phtml` is used for! Using `content/markdown-help_en.phtml` makes sense now.

Reply:
At risk of muddying the waters, there's also the alternative of creating a help page in [the help translations template area](https://github.com/vufind-org/vufind/tree/dev/themes/root/templates/HelpTranslations/en). That might actually make more sense if this is truly a help page rather than general-purpose content. But in the end, the strategy is the same, the only difference is the template file naming pattern and the route to pass to the url helper. :-)

Reply:
Yes, I like that idea. There seems to be no fallback to English though if a help translation is not available, right? Using VuFind in German, I get the message "Sorry, but the help you requested is unavailable in your language".

Reply:
@jschultze, there should still be fallback to English -- the intended behavior is that the "Sorry, but the help you requested is unavailable in your language" message displays as a flash message at the top of the English page when a non-English translation is unavailable. I'm seeing the correct behavior at https://vufind.org/demo -- e.g. set the language to Dutch and then click the top help link in the right side of the page footer.

Reply:
Sorry, I tried it with an empty template. It works as you describe it!

Reply:
I added (basic) help pages in English and German for markdown and emojis and linked them in the notification forms.